RestaurantReservation/Program.cs

```diff
@@ -1,17 +1,33 @@
-﻿using RestaurantReservation.Db;
-using RestaurantReservation.Db.Models.Entities;
-using RestaurantReservation.Db.Models.Enum;
-using RestaurantReservation.Db.Repositories;
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Options;
+using RestaurantReservation.Db;
 
 namespace RestaurantReservation
 {
     class Program
     {
         static async Task Main(string[] args)
         {
-            var context = new RestaurantReservationDbContext();
-            var reservationTester = new ReservationTester(context);
-            await reservationTester.RunTests();
+            var host = Host.CreateDefaultBuilder(args)
+                .ConfigureServices((context, services) =>
+                {
+                    services.AddDbContext<RestaurantReservationDbContext>(options =>
```

Comment:
you have two Program.cs files 
looks like this project isn't now used with the new ASP project correct ?

Reply:
yes that's correct